### PR TITLE
Webos icon fix

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

File: src/components/nowplayingbar/nowplayingbar.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
 
         html += '<div class="nowPlayingBarRight">';
 
-        html += '<button is="paper-icon-button-light" class="muteButton mediaButton"><i class="material-icons volume_up"></i></button>';
+        html += '<button is="paper-icon-button-light" class="muteButton mediaButton"><i class="material-icons"></i></button>';
```
Comment (MrTimscampi) [inline src/components/nowplayingbar/nowplayingbar.js]: I think a better fix would be performed here: https://github.com/jellyfin/jellyfin-web/blob/3b5a63f930b4452dafd04bce57b5875f19c2cece/src/components/nowplayingbar/nowplayingbar.js#L416

Seems like that needs to be changed to remove the volume_up class from the `<i>` and add the other corresponding class.

File: src/components/playback/volumeosd.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ define(['events', 'playbackManager', 'dom', 'browser', 'css!./iconosd', 'materia
     function getOsdElementHtml() {
         var html = '';
 
-        html += '<i class="material-icons iconOsdIcon volume_up"></i>';
+        html += '<i class="material-icons iconOsdIcon"></i>';
```
Comment (MrTimscampi) [inline src/components/playback/volumeosd.js]: Same thing here, I think the proper fix is elsewhere.

File: src/nowplaying.html [hunk 3]
```diff
@@ -63,7 +63,7 @@ <h2 class="nowPlayingPageTitle"></h2>
                         </button>
 
                         <button is="paper-icon-button-light" class="buttonMute autoSize" title="${Mute}">
-                            <i class="xlargePaperIconButton material-icons volume_up"></i>
```
Comment (MrTimscampi) [inline src/nowplaying.html]: This one as well likely needs a JS change instead of a class removal.

File: src/videoosd.html [hunk 4]
```diff
@@ -76,7 +76,7 @@ <h3 class="osdTitle"></h3>
 
                 <div class="volumeButtons hide-mouse-idle-tv">
                     <button is="paper-icon-button-light" class="buttonMute autoSize" title="${Mute} (m)">
-                        <i class="xlargePaperIconButton material-icons volume_up"></i>
```
Comment (MrTimscampi) [inline src/videoosd.html]: Same for this.

File: src/components/nowplayingbar/nowplayingbar.js (hunk 1 above)
Comment (grafixeyehero) [reply src/components/nowplayingbar/nowplayingbar.js]: yap that's why i remove those class b/c it will add  it <i>-here-</i>

**Pull request merged**